cypress/support/fragments/marcAuthority/marcAuthorities.js

```diff
@@ -247,6 +252,8 @@ export default {
   searchBy: (parameter, value) => {
     cy.do(filtersSection.find(searchInput).selectIndex(parameter));
     cy.do(filtersSection.find(searchInput).fillIn(value));
+    // need to wait until value will be applied in case when value is long
+    cy.wait(1000);
```

Comment:
what if we wait in condition if the `value` is long. not to wait for a second in other cases

Reply:
done